packages/x-adapter-platform/src/endpoint-adapters/experience-controls.endpoint-adapter.ts

```diff
@@ -0,0 +1,25 @@
+import { endpointAdapterFactory } from '@empathyco/x-adapter';
+import { ExperienceControlsRequest } from '@empathyco/x-types';
+import { experienceControlsRequestMapper } from '../mappers';
```

Comment:
When importing from the same package, don't import from the barrel. Use the full path.

```suggestion
import { experienceControlsRequestMapper } from '../mappers/requests/experience-controls-request.mapper';
```